modules/RTC/TraceablePeerConnection.js

```diff
@@ -1456,16 +1474,28 @@ TraceablePeerConnection.prototype.setLocalDescription
 };
 
 /**
- * Enables/disables media transmission on this peer connection. When disabled
- * the SDP media direction in the local SDP will be adjusted to 'inactive' which
- * means that no data will be received or sent, but the connection should be
- * kept alive.
- * @param {boolean} active <tt>true</tt> to enable the media transmission or
- * <tt>false</tt> to disable.
+ * Enables/disables audio media transmission on this peer connection. When
+ * disabled the SDP audio media direction in the local SDP will be adjusted to
+ * 'inactive' which means that no data will be sent nor accepted, but
+ * the connection should be kept alive.
+ * @param {boolean} [active] <tt>true</tt> to enable video media transmission or
+ * <tt>false</tt> to disable. If the value is not a boolean the call will have
+ * no effect.
+ * @return {boolean} <tt>true</tt> if the value has changed and sRD/sLD cycle
+ * needs to be executed in order for the changes to take effect or
+ * <tt>false</tt> if the given value was the same as the previous one.
  * @public
  */
-TraceablePeerConnection.prototype.setMediaTransferActive = function(active) {
-    this.mediaTransferActive = active;
+TraceablePeerConnection.prototype.setAudioTransferActive = function(active) {
+    logger.debug(`${this} audio transfer active: ${active}`);
```

Comment:
We use it, because TraceablePeerConnection has 'toString' method implemented for the debugging purpose. We need to know on which peer connection things are happening to figure out whether it's JVB or P2P.